sources/packages/web/src/components/layouts/Institution/sidebar/ManageInstitutionSideBar.vue

```diff
@@ -1,54 +1,59 @@
 <template>
   <v-navigation-drawer app permanent color="background">
-    <v-list density="compact" bg-color="background" active-color="primary">
-      <v-list-item
-        class="sidebar-item"
-        active-class="active-sidebar-item"
-        v-for="item in items"
-        :key="item.label"
-        :to="item.command()"
-        :prepend-icon="item.icon"
-        :title="item.label"
-      />
-    </v-list>
+    <v-list
+      density="compact"
+      bg-color="background"
+      active-color="primary"
+      class="sidebar-item no-wrap"
+      active-class="active-sidebar-item"
+      :items="items"
+    />
   </v-navigation-drawer>
 </template>
 <script lang="ts">
 import { ref } from "vue";
 import { InstitutionRoutesConst } from "@/constants/routes/RouteConstants";
-import { MenuModel } from "@/types";
+import { MenuInterface } from "@/types";
 
 export default {
   components: {},
   setup() {
-    const items = ref<MenuModel[]>([
+    const items = ref<MenuInterface[]>([
```

Comment:
Can we remove the `interface` from the name. If the idea is to create an intermediate class to no conflict with the current `ManuModel` we can call it maybe `ManuItemModel` or just `MenuItem`.